xrdsst/controllers/client.py

```diff
@@ -77,6 +77,24 @@ def import_tls_certs(self):
 
         self.client_import_tls_cert(active_config)
 
+    @ex(help="Unregister client(s)",
+        arguments=[
+            (['--ss'], {'help': 'Security server name', 'dest': 'ss'}),
+            (['--client'], {'help': 'Client(s) Id', 'dest': 'client'})
+        ]
+        )
+    def unregister(self):
+        active_config = self.load_config()
+
+        if self.app.pargs.client is None:
```

Comment:
when checking if command-line parameters are present, I suggest doing it in a similar way as it has been done for example in delete_descriptions() in ServiceController